backend/app/adapter/db/url.go

```diff
@@ -61,6 +63,44 @@ VALUES ($1, $2, $3, $4, $5);`,
 	return err
 }
 
+// UpdateURL updates a URL that exists within the URL table and returns the newly updated URL if there is no error found while
```

Comment:
```suggestion
// UpdateURL updates a URL that exists within the url table.
```
The types on the function signature are very clear.